2025/2025-07-08.md

```diff
@@ -0,0 +1,101 @@
+# 2025-07-08
```

Comment:
This whole file probably wants killing. These minutes have now been merged, so may get a conflict.